src/PsrTarget.php

```diff
@@ -71,7 +71,7 @@ public function export()
                 }
             }
 
-            $this->getLogger()->log($this->_psrLevels[$message[1]], $text, $context);
+            $this->getLogger()->log(isset($this->_psrLevels[$message[1]]) ? $this->_psrLevels[$message[1]] : $message[1], $text, $context);
```

Comment:
That would allow writing anything into level, not only Yii levels and PSR levels. I don't think it is correct.

Reply:
This possibility existed before. There was no verification of the existence of the level. Add it?

Reply:
And how do you think it's correct to make use of the logging levels of the Psr? I wanted to use CRITICAL, for example, but could not in the current implementation.